runtime/lua/vim/lsp/completion.lua

```diff
@@ -503,7 +503,7 @@ local function trigger(bufnr, clients, ctx)
       return
     end
 
-    local matches = {}
+    local matches --[[@type table]] = vim.fn.complete_info({ 'items', 'matches' })['items']
```

Comment:
Is matches needed here? When both items and matches are present, each item will have a match field. However you don't used it. So only items is needed?